src/Components/ApiRequest/ApiRequest.js

```diff
@@ -0,0 +1,11 @@
+const ApiResults = location => {
+  const api = Object.freeze({
+    API_KEY: '&appid=0d26436ded0fcb43c6227b80b9ae1a69',
```

Comment:
you should never commit your API key to a REPOSITORY :) 

Instead of this. please add 
.env file 
where you will store sensitive data.
https://create-react-app.dev/docs/adding-custom-environment-variables

OPEN_WEATHER_API_KEY=''&appid=0d26436ded0fcb43c6227b80b9ae1a69"

then you can use that variable in any file via
process.env.OPEN_WEATHER_API_KEY